docs/user/index.rst

```diff
@@ -143,6 +143,7 @@ and how to write successful documentation.
   :doc:`More guides for authors </guides/authors>`
 
 * **For project administrators**:
+  :doc:`Add A Custom Domain </guides/custom-domains>` |
```

Comment:
Another explicit title here where either the page title doesn't seem right, or we should link to page sections by reference instead.

Reply:
This is a good point - if we follow a pattern where the title is "How to abc" and we want to remove "How to" and just have "abc" in references, then stuff will start drifting when the title becomes "How to abcde".

Reply:
It might likely be that we have to make a choice:

1) Accept variations between titles and navigation entries and try to maintain them
2) Accept to have a lot of "How to..."
3) Stop saying "How to..." in the title, but thus losing the explicit and recommended title practice

Reply:
It occurs to me that a structure of How To guides can be very interesting:

* Title
* Navigation title
* Description (the first paragraph of the page, can appear in `.. seealso::` and SEO description)

I noted it for the future.

Reply:
Mmm yeah, all good points. "How to manage X" is quite clear, and should help with discoverability too, but this does make for some conflict on the front page and in our TOC.

I guess for the front page, I would still link to the top level page on custom domains. The reader is in a very generic spot in our documentation, so pointing them towards generic information on custom domains seems fair.

But we do have some discussion about our end goals for page and navigation structure.